### PR TITLE
DATAUP-667: Fix links in object table

### PR DESCRIPTION
**Pull request by ialarmedalien**
# Description of PR purpose/changes

Ensure that objects that don't appear on the first page of results have the appropriate links.

# Jira Ticket / Issue #

Related Jira ticket: https://kbase-jira.atlassian.net/browse/DATAUP-667
- [x] Added the Jira Ticket to the title of the PR (e.g. `DATAUP-69 Adds a PR template`)

# Testing Instructions
* Details for how to test the PR:
- [x] Tests pass locally and in GitHub Actions
- [x] Changes available by spinning up a local narrative, creating a bulk import cell with more than 50 imports, and checking out the results. You will be amazed!

# Dev Checklist:

- [x] My code follows the guidelines at https://sites.google.com/lbl.gov/trussresources/home?authuser=0
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] (JavaScript) I have run Prettier and ESLint on changed code manually or with a git precommit hook

### CODE REVIEW

File: kbase-extension/static/kbase/js/common/cellComponents/tabs/results/outputWidget.js [hunk 1]
```diff
@@ -119,23 +79,78 @@ define([
...
-                    pageLength: tablePageLength,
+                    data: tableData,
+                    dom: "<'row'<'col-sm-12'tr>><'row'<'col-sm-5'i><'col-sm-7'p>>",
                     lengthChange: false,
-                    fnDrawCallback: () => {
-                        // Hide pagination controls if length is less than or equal to table length
-                        if (arg.objectData.length <= tablePageLength) {
-                            $(container).find('.dataTables_paginate').hide();
+                    pageLength: tablePageLength,
+                    paging: arg.objectData.length > tablePageLength,
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/js/common/cellComponents/tabs/results/outputWidget.js:114]: turn paging on or off, depending on whether it is required ==> don't need the `fnDrawCallback`

File: kbase-extension/static/kbase/js/common/cellComponents/tabs/results/outputWidget.js [hunk 2]
```diff
@@ -119,23 +79,78 @@ define([
...
                     classes: ['kb-panel-container'],
-                    body: renderedOutput.layout,
+                    body: renderOutput(arg.objectData),
                 })
             );
+
             if (arg.objectData.length) {
                 const $objTable = $(ui.getElement('objects-table'));
+
+                const tableData = arg.objectData.map((obj) => {
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/js/common/cellComponents/tabs/results/outputWidget.js:89]: reformat the object data into the appropriate form for the table

File: kbase-extension/static/kbase/js/common/cellComponents/tabs/results/outputWidget.js [hunk 3]
```diff
@@ -119,23 +79,78 @@ define([
...
+                        type,
+                        parsedType,
+                        description,
+                    };
+                });
+
                 $objTable.DataTable({
-                    searching: false,
-                    pageLength: tablePageLength,
+                    data: tableData,
```
Comment (ialarmedalien) [inline kbase-extension/static/kbase/js/common/cellComponents/tabs/results/outputWidget.js:110]: populate the table with the reformatted object data

**Review by briehl — APPROVED** [review]
LGTM.

**Pull request merged**